env/terragrunt.hcl

```diff
@@ -3,14 +3,15 @@ locals {
 }
 
 inputs = {
-  account_id        = "${local.vars.inputs.account_id}"
+  account_id        = local.vars.inputs.env == "local"? "" : "${local.vars.inputs.account_id}"
   billing_tag_key   = "CostCentre"
   billing_tag_value = "forms-platform-${local.vars.inputs.env}"   
-  domain            = "${local.vars.inputs.domain}"
+  domain            = local.vars.inputs.env == "local"? "" :"${local.vars.inputs.domain}"
   env               = "${local.vars.inputs.env}"
-  region            = "ca-central-1" 
+  region            = local.vars.inputs.env == "local" ? "us-east-1":"ca-central-1"
```

Comment:
Similar to the above comment, if you wanted, you could add a `region` var to the `env_vars.hcl` files so that you didn't need the condition:
```hcl
region = "${local.vars.inputs.region}"
```